.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy to gh-pages
+
+on:
+  push:
+    branches:
+      - master
+    #tags:
+    #  - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2.3.4
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: lts/*
+    - run: yarn
+    - run: cd example
+    - run: yarn
+    - run: yarn build
+    - run: yarn push-gh-pages
```

Comment:
this is going to need a `--force` in the script or here because that branch has docs so gh will reject the push, WDYT?

Reply:
Oh yeah good catch.